pkgs/development/interpreters/perl-cross/default.nix

```diff
@@ -36,19 +36,21 @@ let common = {perlVersion, perlSha256}:
       platforms = lib.platforms.all;
     };
   };
-
 in rec {
-
   perl = perl524;
 
-  perl524 = common {
-    perlVersion = "5.24.2";
-    perlSha256 = "1x4yj814a79lcarwb3ab6bbcb36hvb5n4ph4zg3yb0nabsjfi6v0";
-  };
-
   perl522 = common {
-    perlVersion = "5.22.2";
+    perlVersion = "5.22.4";
     perlSha256 = "1yk1xn4wmnrf2ph02j28khqarpyr24qwysjzkjnjv7vh5dygb7ms";
```

Comment:
Does this hash need an update?

Reply:
Good catch!

After investigating, the answer is actually "no"--it's just that the original hash from the earlier commit
was from 5.22.4 not 5.22.2.  This hash matches the one we're using for this version in the 'normal' perl expression.

Reply:
(and I just refetched and confirmed, for some reason the download went realllyyyy slow)